src/services/google-drive-api-service.ts

```diff
@@ -210,60 +252,66 @@ export const getFolderId = async (year: string, month: string): Promise<string>
     console.error(`Error retrieving folder ID for year: ${year}, month: ${month}`, error);
     return null;
   }
-}
+};
 
 /**
  * Gets text content of a specified file in plain text format
- * 
+ *
  * @param file File metadata
  * @returns file's content
  */
 export const getFileText = async (file: File): Promise<string> => {
   try {
     const drive = await getDriveService();
-    const text = (await drive.files.export({
-      fileId: file.id,
-      mimeType: "text/plain"
-    })).data;
+    const text = (
+      await drive.files.export({
+        fileId: file.id,
+        mimeType: "text/plain"
+      })
+    ).data;
     return text as string;
   } catch (error) {
     console.error(`Error retrieving text content of file: ${file.id}`, error);
   }
-}
+};
 
 /**
- * Retrieves the binary content of a single PDF file 
- * 
+ * Retrieves the binary content of a single PDF file
+ *
  * @param file File object
  * @param usedDrive Drive Instance
  * @returns PDF file objects
  */
 export const getFileContentPdf = async (file: File): Promise<PdfFile> => {
   const drive = await getDriveService();
   try {
-    const response = await drive.files.get({
+    const response = await drive.files.get(
+      {
+        fileId: file.id,
+        alt: "media"
+      },
+      { responseType: "stream" }
+    );
+
+    const metadata = await drive.files.get({
       fileId: file.id,
-      alt: "media",
-    },
-    { responseType: "stream"}
-    )
-    if (!response) {
-      console.error(`Failed to fetch file content: ${response.status} - ${response.statusText}`);
-    }
+      fields: "name"
+    });
+
     const pdfBuffer = await streamToBuffer(response.data);
     return {
       id: file.id,
-      name: file.name,
+      name: metadata.data.name,
       content: pdfBuffer
-    }
+    };
   } catch (error) {
     console.error("Error loading the PDF:", error);
   }
-}
+};
 
 /**
  * Creates a new PDF file from binary content and stores it in a specified folder
- * 
+ *
```

Comment:
We should probably remove this createPdfFile and all references as I'm pretty sure we don't want to be creating anything in the drive- read only